packages/ui-components/src/components/range/range.tsx

```diff
@@ -14,7 +14,7 @@ export class KvRange implements IRange, IRangeEvents {
 	/** @inheritdoc */
 	@Prop({ reflect: true }) max: number;
 	/** @inheritdoc */
-	@Prop({ reflect: true }) value?: number = 0;
+	@Prop({ reflect: true, mutable: true }) value?: number = 0;
```

Comment:
As previously discussed, I think we should not mutate this prop and only emit in [`onValueChange`](https://github.com/kelvininc/ui-components/pull/135/files#diff-fffe65616287038846028a9f391b7b789686162c5bc92721fdfce4c956cbcf5cR43).